diff --git a/node_modules/react-hook-form/package.json b/node_modules/react-hook-form/package.json
index 515865a..479e64e 100644
--- a/node_modules/react-hook-form/package.json
+++ b/node_modules/react-hook-form/package.json
@@ -95,7 +95,7 @@
     "react-dom": ">=16.8.0"
   },
   "peerDependencies": {
-    "react": "^16.8.0 || ^17 || ^18 || ^19"
+    "react": "^16.8.0 || ^17 || ^18 || ^19.0.0-rc-f994737d14-20240522"
   },
   "prettier": {
     "printWidth": 80,
diff --git a/node_modules/react-hook-form/dist/index.d.ts b/node_modules/react-hook-form/dist/index.d.ts
index e06f2e8..205a113 100644
--- a/node_modules/react-hook-form/dist/index.d.ts
+++ b/node_modules/react-hook-form/dist/index.d.ts
@@ -1,4 +1,4 @@
-import * as React from 'react';
+import React from 'react';
 
 type FieldValues = Record<string, any>;
 
diff --git a/node_modules/react-hook-form/dist/index.mjs b/node_modules/react-hook-form/dist/index.mjs
index 99b6b74..5037d80 100644
--- a/node_modules/react-hook-form/dist/index.mjs
+++ b/node_modules/react-hook-form/dist/index.mjs
@@ -1,4 +1,4 @@
-import * as e from "react";
+import e from "react";
 import { useRef, useState, useCallback, useMemo } from "react";
 function t(e) {
   let t = e.getBoundingClientRect();
diff --git a/node_modules/react-hook-form/dist/index.cjs b/node_modules/react-hook-form/dist/index.cjs
index 23d548f..489a691 100644
--- a/node_modules/react-hook-form/dist/index.cjs
+++ b/node_modules/react-hook-form/dist/index.cjs
@@ -1,7 +1,7 @@
 "use strict";
 var __create = Object.create;
 var __defProp = Object.defineProperty;
-var __getOwnPropDesc = Object.getOwnPropertyDescriptor;
+var __getOwnPropDesc = Object.getOwnPropertyDescriptor; 
 var __getOwnPropNames = Object.getOwnPropertyNames;
 var __getProtoOf = Object.getPrototypeOf;
 var __hasOwnProp = Object.prototype.hasOwnProperty;
